src/components/Admin/DropDown.jsx

```diff
@@ -1,58 +1,76 @@
 import React, { useState } from "react";
 import Dropdown from "react-bootstrap/Dropdown";
 import { RiArrowDownSLine } from "react-icons/ri";
-const Toggle = ({ children, onClick }) => {
+
+const Toggle = ({ children, onClick, show }) => {
   return (
     <button
       onClick={onClick}
       className="relative z-50 bg-white text-black w-full border-2 rounded-full border-black justify-between flex items-center px-3 py-1"
     >
       {children}
-      <RiArrowDownSLine />
+      <RiArrowDownSLine className={`${show && "rotate-180"} duration-300`} />
     </button>
   );
 };
 const Menu = ({ children, className, setOptions, options }) => {
+  const handleInput = (e) => {
+    setValue(e.target.value);
+    setOptions(
+      options.map((option) => ({
+        ...option,
+        hidden: !option.name.toLowerCase().includes(e.target.value),
+      }))
+    );
+  };
   const [value, setValue] = useState("");
   return (
     <div className={className}>
       <input
         autoFocus
-        className="mx-3 my-2 w-fit ring-0 outline-none"
+        className="mx-3 my-2 w-11/12 ring-0 !outline-0 border-gray-300 border-[1px] text-sm px-2 py-1"
```

Comment:
try outline-none that should solve the outline-0 thing, you shouldn't need the !